ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -134,6 +135,11 @@ public boolean addTransactionHash(final Hash transactionHash) {
     final ValidationResult<TransactionInvalidReason> validationResult =
         validateLocalTransaction(transaction);
     if (validationResult.isValid()) {
+      if (!miningParameters.isMiningEnabled()
```

Comment:
FYI in the merge branch we use MiningParameters enabled/disabled to start and stop the miner at the transition from PoW to PoS.  I think using miningEnabled for this is fine, it just has unintended consequences post-merge.

Reply:
Good point, there's probably a bunch of others. The specific scenario we want to allow is if I'm mining on a PoW network I want to be able to submit 'free' transactions to my pool because I'm going to mine them myself. Maybe we will end up with `isStakingEnabled`?